docs/whitepaper.tex

```diff
@@ -504,60 +504,80 @@ \subsection*{Config UTxO}
 
 \begin{itemize}
-	\item When minting $\sqrt{A_0B_0}$ liquidity tokens on pool open, validation passes.
 	\item When minting a different pool's liquidity token on pool open, validation fails.
 	\item When minting multiple liquidity token types on pool open, validation fails.
 	\item When minting liquidity tokens for a different pool than the pool being spent, validation fails.
 	\item When burning liquidity tokens from a different pool than the pool being spent, validation fails.
 	\item When minting liquidity tokens for a pool at an address that is not the Pool Address Validator, validation fails.
-	\item Given a pool containing total issued liquidity tokens of quantity $t$ of $T$ and asset
-	      pair of quantities $a$ of $A$ and $b$ of $B$, when the user provides quantities $\Delta a$ and $\Delta b$
-  	      and mints a quantity of liquidity tokens less than or equal to
-	      $T \cdot \frac{\sqrt{ (a + \Delta a) \cdot (b + \Delta b)} - \sqrt{a \cdot b}}{\sqrt{a \cdot b}}$
-	      , validation passes.
-	\item Given a pool containing total issued liquidity tokens of quantity $t$ of $T$ and asset
-	      pair of quantities $a$ of $A$ and $b$ of $B$, when the user provides quantities $\Delta a$ and $\Delta b$
-	      and mints a quantity of liquidity tokens greater than
-              $T \cdot \frac{\sqrt{ (a + \Delta a) \cdot (b + \Delta b)} - \sqrt{a \cdot b}}{\sqrt{a \cdot b}}$
-	      , validation fails.
-	\item Given a pool containing total issued liquidity tokens of quantity $t$ of $T$ and asset pair of quantities
-	      $a$ of $A$ and $b$ of $B$, when the user burns
-	      their liquidity token quantity $\Delta t$ and receives a quantity of $A$ less than or equal to
-	      $ a \cdot \Delta t/t $ and quantity of $B$ less than or equal to $ a \cdot \delta t/t $
-	      , validation passes.
-	\item Given a pool containing total issued liquidity tokens of quantity $t$ of $T$ and asset
-	      pair of quantities $a$ of $A$ and $b$ of $B$, when the user burns
-	      their liquidity token quantity $\Delta t$ and receives a quantity of $A$ greater than
-	      $ a \cdot \Delta t/t $ or (||) quantity of token $B$ greater than or equal to $ a \cdot \delta t/t $
-	      , validation fails.
 \end{itemize}
 
 \subsection*{Pool id Token Minting Policy tests}
 
+For this section a "valid pool open" refers to a Tx where
+
+\begin{itemize}
+	\item A pool is opened with 100 each of two dummy tokens.
+	\item The admin fees are set to 0
+	\item the issued liquidity is $\sqrt{A_0 B_0}$ (in this case 100)
+	\item the pool liveness is set to true
+	\item The transaction should reference the valid config utxo
+	\item A seed utxo should be chosen
+	\item The seed utxo is spent
+	\item The seed utxo is passed to the pool id minting policy via the redeemer
+	\item the pool id is the hash of the seed utxo
+	\item The same number as the issued liquidity of liquidity tokens of the same id as the pool are minted
```

Comment:
```suggestion
	\item The number of issued liquidity tokens set in the pool's UTxO datum matches the number of liquidity tokens minted from the liquidity token minting policy.
```

Just an idea about how this could be improved.